packages/mobile/src/account/Settings.tsx

```diff
@@ -329,6 +329,24 @@ export class Account extends React.Component<Props, State> {
     this.setState({ showRevokeModal: false })
   }
 
+  goToChangePin = () => {
+    ValoraAnalytics.track(SettingsEvents.change_pin_start)
+    ensurePincode()
+      .then((pinIsCorrect) => {
+        if (pinIsCorrect) {
+          ValoraAnalytics.track(SettingsEvents.change_pin_current_pin_entered)
+          this.props.navigation.navigate(Screens.PincodeSet, {
```

Comment:
Nit: Can you please use the `navigate` function from `NavigationService` instead of calling it from props? I know that's not how the rest of the file has it but that's because this file is old and in need of a refactor :)

The benefit is that the function has built in type checking + logging so it's preferable.

Reply:
sure! sounds good